src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -58,6 +58,9 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ElbControllerImpl.class);
 
+    private ELBApi elbApi;
+    private LoadBalancerServiceContext loadBalancerServiceContext;
```

Comment:
the idea was that we could reuse the elbApi if it existed already, rather than having to create it each time, but you think put it back to local variables?